incubator/hnc/config/samples/hnc_v1alpha1_anchor.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
Did you add this manually?

Reply:
Yes this file is added manually to be used in `kubectl apply -f` to test v1a1 CRs in 0.5 can still work after CRD conversion. It's commented in the commit message test part. I will add a comment here too.

Reply:
thanks!